codes/go/chapter_stack_and_queue/deque.go

```diff
@@ -0,0 +1,27 @@
+// File: deque.go
+// Created Time: 2022-11-29
+// Author: Reanon (793584285@qq.com)
+
+package chapter_stack_and_queue
+
+type Deque interface {
```

Comment:
Testing for the Go built-in deque is required.

Reply:
Do you use `interface` to reveal the usage of built-in Stack, Queue, and Deque?
I'm wondering if it is the only way as It is not consistent with other languages.